MotionMark/resources/statistics.js

```diff
@@ -249,17 +253,22 @@ Regression = Utilities.createClass(
             };
         }
 
+        // Sort by increasing complexity.
+        var sortedSamples = samples.slice().sort((a, b) => a[complexityIndex] - b[complexityIndex]);
```

Comment:
We sort the `samples` out of place after making a copy of it. Can we make the caller sort the array (in place if possible) and pass the sorted array instead?

Reply:
I think Regression should be self-contained; the caller should be allowed to pass unsorted data.